requirements.txt

```diff
@@ -1,4 +1,4 @@
-karton-core==4.1.0
+karton-core>=4.2.0
```

Comment:
I'm not sure if we don't want to also include an upper boundary like `<5.0.0` cc @psrok1

Reply:
Yes, added suggestion below :+1: